assets/prototype/domain/shared/entities/tickets/predicates/isArchived.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { Ticket } from '../../../../eventEditor/data/types';
+
+/**
+ * @function
+ * @param {Object} ticket object
+ * @return {boolean} true if ticket is archived
+ */
+const isArchived = (ticket: Ticket): boolean => ticket.isDeleted;
```

Comment:
this will need fixing after rebasing branch onto EDTR/master

Reply:
replaced `isDeleted` with `isTrashed`